src/Selector/Xpath/Manipulator.php

```diff
@@ -66,4 +66,50 @@ public function prepend($xpath, $prefix)
 
         return implode(' | ', $expressions);
     }
+
+    /**
+     * Splits the XPath into parts that are separated by the union operator.
+     *
+     * @param string $xpath
+     *
+     * @return string[]
+     */
+    private function splitUnionParts($xpath)
+    {
+        // Split any unions into individual expressions. We need to iterate
+        // through the string to correctly parse opening/closing quotes and
+        // braces which is not possible with regular expressions.
+        $unionParts = array();
+        $singleQuoteOpen = false;
+        $doubleQuoteOpen = false;
```

Comment:
I would name them ``$inSingleQuotedString`` and ``$inDoubleQuotedString``. It would better reflect their usage